app/components/listing/MapOfLocations.tsx

```diff
@@ -40,8 +40,8 @@ export const MapOfLocations = ({
           {locations.map(({ address, id }, i) => (
             <CustomMarker
               key={id}
-              lat={address?.latitude || 0}
-              lng={address?.longitude || 0}
+              lat={Number(address?.latitude) || 0}
+              lng={Number(address?.longitude) || 0}
```

Comment:
I don't love this inline conversion because it will make the code easier to reason about if we can stabilize the type of these values closer to where we fetch them.